cli/wallet/wallet.go

```diff
@@ -487,6 +492,8 @@ func transferAsset(ctx *cli.Context) error {
 		return cli.NewExitError(fmt.Errorf("wallet contains no account for '%s'", from), 1)
 	}
 
+	remark14 := ctx.String("remark14")
+
```

Comment:
What about declaring it right where it is used?

Reply:
Usually I prefer gathering all the data needed first and then doing something with it.